reference/functions/packagesmatching.markdown

```diff
@@ -39,6 +39,8 @@ The return is a data container with a list of package descriptions, looking like
 The following code extracts just the package names, then looks for
 some desired packages, and finally reports if they are installed.
 
+*IMPORTANT:* Please note that you need to provide ```package_inventory``` attribute in ```body common control``` in order to be able to use this function. Also depending on the value(s) of ```package_inventory``` only packages from selected package modules will be returned. For more information about ```package_inventory``` please read [`package_inventory`][Components and Common Control#package_inventory] section.
```

Comment:
Double asterisks for bolding in markdown.
**IMPORTANT:**

Single backticks for verbatium or automatic linking to cfengine attributes:

`package_inventory` `body common control`